### PR TITLE
update scripts/docker/zeppelin/bin/Dockerfile to JDK11

### PR DESCRIPTION
**Pull request by flowy0**
#### What is this PR for?
To update `zeppelin/scripts/docker/zeppelin/bin/Dockerfile` to OpenJDK 11

Follow up PR from https://github.com/apache/zeppelin/pull/4751 and https://github.com/apache/zeppelin/pull/4752

### What type of PR is it?
Bug Fix

### What is the Jira issue?
* [ZEPPELIN-6011](https://issues.apache.org/jira/browse/ZEPPELIN-6011)

### How should this be tested?
* Outline any manual steps to test the PR here.
  * Not too sure how to automate this testing as i'm new at this.

### CODE REVIEW

**Comment by jongyoul** [comment]
Thank you for your contribution. Could you please change it as well?

https://github.com/apache/zeppelin/blob/fa6e3ee3b83aa5ffc070557780ec111386b8c951/scripts/docker/spark-cluster-managers/spark_yarn_cluster/Dockerfile#L37

**Comment by flowy0** [comment]
> Thank you for your contribution. Could you please change it as well?
> 
> https://github.com/apache/zeppelin/blob/fa6e3ee3b83aa5ffc070557780ec111386b8c951/scripts/docker/spark-cluster-managers/spark_yarn_cluster/Dockerfile#L37

i'm not too familiar with yarn/centos. I did try to amend this, but it did seem like other urls required updates as well. The docker image did not build.

**Comment by pan3793** [comment]
seems the minconda version also needs to be updated?

File: scripts/docker/zeppelin/bin/Dockerfile [hunk 1]
```diff
@@ -24,22 +24,23 @@ ENV LOG_TAG="[ZEPPELIN_${Z_VERSION}]:" \
...
     apt-get -y update && \
     # Switch back to install JRE instead of JDK when moving to JDK9 or later.
-    DEBIAN_FRONTEND=noninteractive apt-get install -y locales language-pack-en tini openjdk-8-jdk-headless wget unzip && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y locales language-pack-en tini openjdk-11-jdk-headless wget unzip && \
     # Cleanup
     rm -rf /var/lib/apt/lists/* && \
     apt-get autoclean && \
     apt-get clean
 
+
```
Comment (pan3793) [inline scripts/docker/zeppelin/bin/Dockerfile:39]: nit: unnecessary change

**Comment by flowy0** [comment]
Just edited the file.

**Review by jongyoul — APPROVED** [review]

**Comment by jongyoul** [comment]
@flowy0 Thank you for your contribution. I already gave you a credit in the JIRA as well. By the way, I missed changing the title so please check if you are in a list of contributors or not when releasing new versions.
